src/powerman/client.c

```diff
@@ -1105,9 +1106,9 @@ void cli_post_poll(xpollfd_t pfd)
                 err(false, "client poll: hangup");
             if (flags & XPOLLNVAL)
                 err(false, "client poll: fd not open");
-            if (flags & (XPOLLERR | XPOLLHUP | XPOLLNVAL))
+            if (flags & (XPOLLERR | XPOLLNVAL))
                 goto client_dead;
-            if ((flags & XPOLLIN))
+            if ((flags & XPOLLIN) || (flags & XPOLLHUP))
```

Comment:
for consistent style, `flags & (XPOLLIN | XPOLLHUP)`?

Reply:
Ah good one, thanks.